packages/editor/src/components/global-styles/hooks.js

```diff
@@ -188,8 +188,8 @@ export function useGlobalStyles() {
 /**
  * Hook to get a style value from global styles
  *
- * @param {string} path      Style path (e.g., 'color.background')
- * @param {string} blockName Optional block name
+ * @param {string}  path      Style path (e.g., 'color.background')
+ * @param {string=} blockName Optional block name
```

Comment:
Tiniest of tiny nits: is this meant to be a `?`? (Same with the doc comment for `useSetting`)

Reply:
Good to know, thanks! TIL